grafana/provisioning/dashboards/dev/anchorservice.json

```diff
@@ -2901,12 +3216,12 @@
         ]
     },
     "time": {
-        "from": "now-12h",
```

Comment:
Why 7 day? Would be more helpful to keep it consistent I think so that when we look at the different dashboards it is obvious the time range

Reply:
right!  that was a mistake, I used 7d because the anchor workers are so infrequent in dev but didn't mean to save it, fixed